src/test/java/api/requests/PickSlipsTests.java

```diff
@@ -35,326 +40,356 @@
 public class PickSlipsTests extends APITests {
   private static final String TOTAL_RECORDS = "totalRecords";
   private static final String PICK_SLIPS_KEY = "pickSlips";
-  private static final String ID_KEY = "id";
-  private static final String HOLDINGS_RECORD_ID_KEY = "holdingsRecordId";
-  private static final String INSTANCE_ID_KEY = "instanceId";
-  private static final String TITLE_KEY = "title";
-  private static final String BARCODE_KEY = "barcode";
-  private static final String CALL_NUMBER_KEY = "callNumber";
-  private static final String STATUS_KEY = "status";
-  private static final String CONTRIBUTORS_KEY = "contributors";
-  private static final String LOCATION_KEY = "location";
-  private static final String NAME_KEY = "name";
-  private static final String CODE_KEY = "code";
+  private static final String ITEM_KEY = "item";
+  private static final String REQUEST_KEY = "request";
+  private static final String REQUESTER_KEY = "requester";
 
   @Test
-  public void reportIsEmptyForNonExistentServicePointId() {
-
-    final UUID servicePointId = servicePointsFixture.cd1().getId();
-    final InventoryItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+  public void responseIsEmptyForNonExistentServicePointId() {
```

Comment:
The assertion appears to check whether the report contains any items, rather than for no response from the API.

Why has the name been changed from `reportIsEmpty` to `responseIsEmpty`?

Reply:
> The assertion appears to check whether the report contains any items, rather than for no response from the API.

That was exaclty my intention. Is the name misleading?

Initially the name of the whole feature was PickSlips**Report**, but then it [was decided](https://issues.folio.org/browse/CIRC-494?focusedCommentId=66032&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-66032) that this is not a **report**. The names of tests are a residue from that first implementation.

Reply:
> > The assertion appears to check whether the report contains any items, rather than for no response from the API.
> 
> That was exaclty my intention. Is the name misleading?

When I read an API test name that says the response should be empty, I interpret that as the HTTP response having no body (usually a 204 status code), rather than there are no pick slips in the response.

> Initially the name of the whole feature was PickSlips**Report**, but then it [was decided](https://issues.folio.org/browse/CIRC-494?focusedCommentId=66032&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-66032) that this is not a **report**. The names of tests are a residue from that first implementation.

I was not aware of that decision. If it isn't a report, what is this API?

Reply:
Fair enough. I renamed the tests. Hope they make more sense now.

When I created my first pull request on pick slips, you raised your concerns that PickSlipReport does not appear to be a "report" in FOLIO terms, meaning that it does not resemble other reports we have. What it does is that it creates a list of pick slips on demand.

Reply:
Ok, thanks. I'll resole this, and review the new names